packages/starknet-snap/src/createAccount.ts

```diff
@@ -148,6 +149,10 @@ export async function createAccount(params: ApiParams, silentMode = false) {
 
       logger.log(`createAccount:\ndeployResp: ${toJson(deployResp)}`);
 
+      if (waitMode) {
```

Comment:
Can you add a comment here describing when we would `wait` and when we would not ?

Reply:
sure, the wait mode basically is only use when you need other txn compose together, and to make sure this txn execute complete, the wait mode is something that we need

Reply:
added